core/arch/asm_defines.asm

```diff
@@ -411,6 +419,24 @@ ASSUME fs:_DATA @N@\
 #   define ARG8_NORETADDR  QWORD [7*ARG_SZ + REG_XSP]
 #   define ARG9_NORETADDR  QWORD [8*ARG_SZ + REG_XSP]
 #   define ARG10_NORETADDR QWORD [9*ARG_SZ + REG_XSP]
+#   define PUSH_CALLEE_SAVED_REGS() \
```

Comment:
No definition for AARCHXX?  Won't drreg-test then fail to build on arm?

Reply:
The arm version of the test for some reason doesn't emulate the ABI (no push/pop). Not sure why, and I don't have any way to test. Do you want me to add it anyway?

Reply:
Either way -- should prob have a comment `XXX i#3289` for adding it or sthg if not.

Reply:
Done